docs/user-roles.rst

```diff
@@ -44,3 +57,63 @@ also perform every available operation inside that project.
     organization.
 
 A *project member* has read-only access to the project.
+
+
+.. _user-ops:
+
+User operations
+===============
+
+This section describes issues relating to user operations that derive from the
```

Comment:
I don't think it is very positive to talk about "issues" in the first sentence. This paragraph could state that there are different types of database users and that their differences are described in more detail below.

Reply:
That's a great idea! It changes the nature of the doc a bit but I think for the better.